packages/framework/aqueduct/src/data-object-factories/pureDataObjectFactory.ts

```diff
@@ -59,12 +59,12 @@ async function createDataObject<
 	// request mixin in
 	runtimeClass = mixinRequestHandler(
 		async (request: IRequest, runtimeArg: FluidDataStoreRuntime) => {
-			const router: FluidObject<IFluidRouter> = await runtimeArg.entryPoint.get();
+			const router = (await runtimeArg.entryPoint.get()) as TObj;
 			assert(
-				router.IFluidRouter !== undefined,
-				0x469 /* Data store runtime entryPoint is not an IFluidRouter */,
+				router.request !== undefined,
```

Comment:
Don't do it but it's tempting to update this to be `router instanceof(PureDataObject)`